kili/cli/helpers.py

```diff
@@ -4,9 +4,10 @@
 import warnings
 from typing import Any, Dict, List, Optional
 
+from kili.client import Kili
 from kili.graphql_client import GraphQLClientName
 
-from ..client import Kili
```

Comment:
Can we rather fix the associated warnings or is it too hard ?

Reply:
I agree, this one is fixable